frontend/src/components/CourseModal/CourseModal.tsx

```diff
@@ -255,7 +250,7 @@ function CourseModal() {
       >
         <PopoutSelect<Option, false>
           value={sectionsOptions.get(listing.section)}
-          options={sectionsOptions.values().toArray()}
+          options={[...sectionsOptions.values()]}
```

Comment:
Check https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Iterator/toArray; its support is too low, so you need to either install a polyfill from core-js (I don't think Vite automatically polyfills) or just use `[...]` which I did here